src/handlers/suggest/handler.js

```diff
@@ -1,90 +1,178 @@
 const config = require('../../../config.json');
-const request = require('request'); // request is deprecated, but we depend on it through other packages, so we might as well use it.
-const jsonDiff = require('json-diff');
 const { Octokit } = require('@octokit/rest');
-const octokit = new Octokit({
+const { createPullRequest } = require('octokit-plugin-create-pull-request');
+
+const myOctokit = Octokit.plugin(createPullRequest);
+const octokit = new myOctokit({
     auth: config.suggest.github_token,
     userAgent: 'Datenanfragen.de suggest-api',
 });
 
-/**
- * Returns `JSON.stringify()`'s output for a new entry, otherwise it returns a diff by downloading the old entry from `raw.github.com`.
- * @param request_body
- * @returns {Promise<string|*>}
- */
-async function getMessageContent(request_body) {
-    if (request_body.new) return JSON.stringify(request_body.data, null, 4);
-
-    // an update was suggested, lets make a diff!
-    const url = `https://raw.githubusercontent.com/${config.suggest.owner}/${config.suggest.repo}/${config.suggest.branch}/companies/${request_body.data.slug}.json`;
-
-    // wrap request in Promise, see https://stackoverflow.com/a/51162901
-    const old_json = await new Promise((resolve, reject) => {
-        request({ uri: url }, function (error, response, body) {
-            if (error || response.statusCode !== 200) {
-                console.error(error);
-                console.error(`Responsecode: ${response.statusCode}`);
-                reject(`Request to ${url} failed`);
-            }
-            resolve(body);
-        });
-    });
-    return jsonDiff.diffString(JSON.parse(old_json), request_body.data, { color: '' });
-}
-
-async function messageBody(request_body) {
-    return `The following suggestion was submitted through the website:
+async function suggest(request, h) {
+    let company = request.payload.data;
 
-\`\`\`${request_body.new ? '' : 'diff'}
-${await getMessageContent(request_body)}
-\`\`\`
+    const file_path = 'companies/' + company.slug + '.json';
+    const files = {};
+    files[file_path] = JSON.stringify(company, null, 4) + '\n';
 
-**[Edit](https://company-json.netlify.com/#!doc=${encodeURIComponent(JSON.stringify(request_body.data))})**`;
-}
+    const title = request.payload.new
+        ? 'New company suggestion' + (company.name ? ': `' + company.name + '`' : '')
+        : 'Suggested update for company `' + company.slug + '`';
+    const commit_msg =
+        (request.payload.new
+            ? 'Add ' + (company.name ? ': `' + company.name + '`' : '')
```

Comment:
I really don't like having the backticks in the commit messages. I would much prefer double or single quotes.

Also, new companies should also have ` (community contribution)` in the commit message.